### ISSUE TITLE
Pulsatile liver

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10266]
**Preferred term label:**
Pulsatile liver

**Synonyms**
Cardiac synchronous pulsation on palpation

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Pericarditis, constrictive
Aortic or tricuspid regurgitation

**Your nano-attribution (ORCID)**

**Comment by MickeySegal** [comment]
Login.gov is now back up, and the UMLS CUI is C0277926

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000039
- label: Pulsatile liver
- parents: [Abnormal liver physiology (HP:0031865)](https://hpo.jax.org/app/browse/term/HP:0031865)
- pmid: PMID:8427187
- def: Cardiac synchronous pulsation on palpation.
- comment: A pulsating liver is characteristic of significant tricuspid regurgitation. Systolic pulsations of the liver in patients with primary severe tricuspid regurgitation are due to transmission of a prominent right atrial v wave.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10266
- synonyms:
